installscripts/jazz-terraform-unix-noinstances/commands.tf

```diff
@@ -62,9 +62,13 @@ variable "injectingBootstrapToJenkinsfiles_cmd" {
   type = "string"
   default = "./scripts/injectingToJenkinsfile.sh"
 }
+variable "configureJenkinsCE_cmd" {
+  type = "string"
+  default = "./scripts/configure-cli.sh"
+}
 variable "launchJenkinsCE_cmd" {
   type = "string"
-  default = "../dockerfiles/jenkins-ce/chefclient.sh"
+  default = "../dockerfiles/jenkins-ce/configurejenkins.sh"
```

Comment:
There is no script with this name at this location.

Reply:
Done